packages/vue/src/components/molecules/SfBanner/SfBanner.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <section class="sf-banner" :style="style" v-on="isMobile ? $listeners : {}">
-    <div class="sf-banner__container">
+    <component :is="tag" class="sf-banner__button-wrapper" :link="link">
```

Comment:
if it's not only a button tag I'd change also its class name - what do you think about "wrapper" ? :)

Reply:
@justyna-13 fixed that, take a look please